pyEpiabm/pyEpiabm/core/household.py

```diff
@@ -65,3 +68,29 @@ def add_person(self, person):
         """
         self.persons.append(person)
         person.household = self
+        if person.infection_status == InfectionStatus.Susceptible:
+            self.add_susceptible_person(person)
+
+    def add_susceptible_person(self, susceptible_person):
+        """Adds a person to the list of susceptible people in the household.
+
+        Parameters
+        ----------
+        susceptible_person : Person
+            Person to be added
+
+        """
+        if susceptible_person not in self.susceptible_persons:
```

Comment:
Because we are using a set, there is no need to check membership before adding. You can just call add with that person, if they are already there, it will have no effect.

Reply:
No longer using a set

Reply:
I think we should discuss this further - if we are just using the list to check inclusion in a group of unique elements then the set is appropriate and we shouldn't try to implement this ourselves using lists. We can look into why the functional tests are failing but it is likely because we're misusing this list!